app/app.go

```diff
@@ -49,11 +62,11 @@ func Run(cfg *config.Config) error {
 
 	// Setup DB connection
 	switch strings.ToLower(Config.DatabaseType) {
-	case "cassandra":
+	case cassandraName:
 		var err error
 		db, err = cassandra.Dial(Config.Databases.Cassandra)
 		if err != nil {
-			return fmt.Errorf("Unable to connect to cassandra database - %s", err)
+			return fmt.Errorf("%s %s- %s", unableConnectDB, cassandraName, err)
```

Comment:
Also here, same question.

Reply:
same thing :D

Reply:
but also if we add more DBs then we can use this in separate function and call that

Reply:
let's skip adding these for now, maybe they will be useful in the future but right now I'm not sure they are needed.

Reply:
ok